ui_components/src/village/Village.js

```diff
@@ -821,7 +821,7 @@ function KageQuarters({
         else {
             setModalState("confirm_policy");
             setModalHeader("Confirmation");
-            setModalText("Are you sure you want to change policies? You will be unable to select a new policy for 14 days.");
+            setModalText("Are you sure you want to change policies? You will be unable to select a new policy for 3 days.");
```

Comment:
Could we make this reference the constant, so we don't have to manually update if we change policy cooldown time in the future?

Reply:
Yeah I think we need to pass this with the initial query load, will tweak.